dex/src/main/scala/com/wavesplatform/dex/api/http/entities/MatcherResponse.scala

```diff
@@ -85,7 +85,7 @@ case class OrderCancelRejected(error: MatcherError) extends MatcherResponse(C.Ba
 case object InvalidSignature extends MatcherResponse(C.BadRequest, error.RequestInvalidSignature)
 case class NotImplemented(error: MatcherError) extends MatcherResponse(C.NotImplemented, error)
 case class CanNotPersist(error: MatcherError) extends MatcherResponse(C.ServiceUnavailable, error)
-case class OrderBookUnavailable(error: MatcherError) extends MatcherResponse(C.ServiceUnavailable, error)
+case class OrderBookUnavailable(error: MatcherError) extends MatcherResponse(C.BadRequest, error)
```

Comment:
`OrderBookUnavailable` is used not only when we are trying to delete an order book, e.g. it was stopped by internal reasons and we can't start it. In these cases `ServiceUnavailable` is appropriate. See usages of `OrderBookUnavailable`.

So I think we need:
1. A new class, or just use `SimpleErrorResponse` with `BadRequest`
2. Use it only for order book deletion (see `MatcherActor`)